modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/cards/VerticalCard.js

```diff
@@ -118,6 +119,7 @@ export default function VerticalCard({
 	return (
 		<ClayCardWithInfo
 			actions={normalizedActions}
+			aria-label={titleAriaLabel}
```

Comment:
I have 2 questions here:
* We'd like initial SSR markup we send to the browser to include the `aria-label` as well (in `processStartTag()`) so that the accessibility tree does not change after react hydration
* There is one usage where `aria-label` is given to the tag via attribute, together with the vertical card model ([code](https://github.com/liferay/liferay-portal/blob/65fbbbb5e44c5b5fb6935dcac984dca0d957c035/modules/apps/item-selector/item-selector-web/src/main/resources/META-INF/resources/view_item_selector_view_descriptor.jsp#L97-L101)). In cases like this, tag should control what to send as as `aria-label`. This makes me believe that the naming should be unified (`titleAriaLabel` change to `ariaLabel`)

@markocikos your thoughts here are welcome 
cc @edalgrin

Reply:
Hi @dsanz! Thanks for the feedback 😄 Answering inline:
> 
> * We'd like initial SSR markup we send to the browser to include the `aria-label` as well (in `processStartTag()`) so that the accessibility tree does not change after react hydration. Would you mind adding this?

I added another commit doing this!

> * There is one usage where `aria-label` is given to the tag via attribute, together with the vertical card model ([code](https://github.com/liferay/liferay-portal/blob/65fbbbb5e44c5b5fb6935dcac984dca0d957c035/modules/apps/item-selector/item-selector-web/src/main/resources/META-INF/resources/view_item_selector_view_descriptor.jsp#L97-L101)). In cases like this, tag should control what to send as as `aria-label`. This makes me believe that the naming should be unified (`titleAriaLabel` change to `ariaLabel`)
> 

Regarding this, the thing is that they are different things. The card itself can have an `aria-label` (that's the one that can be passed to the tag via attribute, like in the example you attached), but what I'm adding is an `aria-label` specifically for the card title. And both can be used at the same time for different purposes. What I also think is that maybe we could have called the param of the Clay component `titleAriaLabel` as well, to avoid confusion, but that's something they should change in Clay in case we want to (cc @matuzalemteles).

Reply:
I thought about this option also to add `titleAriaLabel` instead of using `aria-label` the thought is that we already had the API of `aria-label` which is rendered in the root element of the components but has no use because these elements don't has focus because they are just presentational, so the title cards were used to be the `aria-label` API since they are the text that tells "what is", so basically we just moved to where the `aria-label ` is rendered in all Card components.

I would say that we don't need `titleAriaLabel` here because the `aria-label` API already exists because it is a native property for the element in this case the element would be the entire context of the Card component but we just say which element inside the component we render this property.

Reply:
Hey @matuzalemsteles, you are suggesting that we pass the `aria-label` directly when rendering the `clay:vertical-card` tag, instead of passing it in the Java context, right?

At this moment, if I do that, the `aria-label` is added to the Card itself, instead of to the title. Is that because the last Clay release was reverted?

Reply:
@sandrodw3 yeah, unfortunately we had to revert the changes from the last release of Clay due to some MultiSelect regressions that caused a lot of problems, I'm working on it now to bring the version back to DXP.

Reply:
Ok! In that case I'll close this PR and wait for the release to be available again, thanks Matu 😄

Reply:
Sure no problem, sorry for that. I'll let you know when I send it to master, I should send something today.

Reply:
Perfect! Thanks a lot Matu